spec/injection_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'injection'
+
+describe Array do
+
+  describe 'injection' do
+    subject{[1,2,3]}
+
+    it 'should take subject and block and return the value of the block ' do
+      expect(subject.injection{|x,y| x + y}).to eq 6
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.